RecoVertex/VertexPrimitives/src/TransientVertex.cc

```diff
@@ -26,9 +26,21 @@ TransientVertex::TransientVertex(const GlobalPoint & pos, const GlobalError & po
   //  addTracks(tracks);
 }
 
+TransientVertex::TransientVertex(const GlobalPoint & pos, const double time,
+                                 const GlobalError & posTimeError,
+                                 const std::vector<TransientTrack> & tracks, float chi2) :
+  theVertexState(pos, time, posTimeError), theOriginalTracks(tracks),
+  theChi2(chi2), theNDF(0), vertexValid(true), withPrior(false),
+  theWeightMapIsAvailable(false), theCovMapAvailable(false), 
+  withRefittedTracks(false)
+{
+  theNDF = 2.*theOriginalTracks.size() - 3.;
+  //  addTracks(tracks);
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant.

Looks like it's copy-pasted from all other alternative implementations.
Better not reproduce these 9 yo left out edits.